cluegonauts/clueless/classes.py

```diff
@@ -59,3 +59,187 @@ def serialize_selected(self) -> List[str]:
         Serialize the selected characters
         """
         return [char.id for char in self.characters if char.selected]
+    
```

Comment:
I think it should be fine to just delete these instead of commenting out, we can always find them in Aastha's old PR if needed.